iphone/TitaniumKit/TitaniumKit/Sources/API/KrollBridge.m

```diff
@@ -34,6 +34,26 @@
 
 @implementation KrollBridge
 
+- (NSDictionary<NSString *, id> *)modules
+{
+  // This is for backwards compatibility with native modules relying on a modules property that
+  // cached module ids -> module instances
+  // Note that this one will only return external native modules and relies on their moduleId
+  // method returning the correct valiue
+  NSDictionary<NSString *, id> *orig = [self.host valueForKey:@"modules"];
+  NSMutableDictionary<NSString *, id> *copy = [orig mutableCopy];
```

Comment:
I think you need to do the following here or else a memory leak will occur.
```objc
[[orig mutableCopy] autorelease];
```

Reply:
Instead of autorelease here, generally we should do before while return. Line 54 should be  -
`return [copy autorelease]`